src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -62,5 +156,33 @@ private static UIElement CreateElement(string text, double lineHeight, TextForma
             border.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
             return border;
         }
+
+        /// <summary>
+        /// Determines if the border is being moused over and shows the info accordingly
+        /// </summary>
+        private void Border_ToolTipOpening(object sender, ToolTipEventArgs e)
+        {
+            var border = (Border)sender;
+            e.Handled = true;
+
+            bool KeepOpen()
+            {
+                var mousePoint = Mouse.GetPosition(border);
+                return !(mousePoint.X > border.ActualWidth || mousePoint.X < 0 || mousePoint.Y > border.ActualHeight || mousePoint.Y < 0);
+            }
+
+            var toolTipPresenter = _toolTipService.CreatePresenter(_textView, new ToolTipParameters(true, false, KeepOpen));
```

Comment:
Use named parameters for the true/false (or build your feature into GitHub -- your choice!)

Reply:
Hmmm I'll talk to the Github team and get back to you